content/guides/mobile/android/install.md

```diff
@@ -0,0 +1,52 @@
+---
+related_pages:
+ - sdks-and-tools
+---
+
+# Android SDK Installation
+
+The Android SDK may be obtained through several methods:
+
+* Adding as a Maven or Gradle dependency
+* Cloning the source into your project
+* Downloading one of the precompiled JARs
```

Comment:
Does the SDK itself have further instructions? Is it worth linking there? Maybe use a CTA block at the end of the page?

Reply:
Linked to the Github repo readme